mdpow/tests/test_Gsolv.py

```diff
@@ -2,26 +2,25 @@
 import os
 import pybol
 import numpy as np
+import gromacs
 from gromacs.utilities import in_dir
+from mdpow.fep import *
+from mdpow.equil import *
```

Comment:
avoid blanket imports — only import what you need or better, use the qualified names as before